packages/utils/functions/fetch/baseAppFetch/index.ts

```diff
@@ -1,5 +1,6 @@
 import humps from 'humps'
```

Comment:
This has all been taken care of in the AllAuthApi class. You should be using that instead.